portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting all quotes in Datastore. */
+@WebServlet("/deleteQuote")
+public class DeleteDataServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Quote");
+    PreparedQuery quotes = datastore.prepare(query);
+
+    for (Entity quoteEntity : quotes.asIterable()) {
+      Key quoteEntityKey = quoteEntity.getKey();
+      datastore.delete(quoteEntityKey);
```

Comment:
It's not that important, but you can also pass a `List<Key>` to the `datastore.delete(...)` method and then they'll be deleted in a single call to the datastore service instead of individual calls for each entity which should be much more efficient. Maybe build a `List` of the keys and then pass them all at the same time?

Reply:
Done.